### PR TITLE
Updated required fields

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7141)
When working on the add-child workflow we missed a few required fields and we need to make these fields required.

## Testing done
All unit tests updated accordingly and pass

## Screenshots
Screen Shot 2020-03-25 at 1 35 05 PM

## Acceptance criteria
- [x] Update the required fields

### CODE REVIEW

File: src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-place-of-birth/child-place-of-birth.js [hunk 1]
```diff
@@ -74,39 +74,34 @@ export const uiSchema = {
...
-          {
-            'ui:options': {
-              expandUnder: 'stepchild',
-              expandUnderCondition: true,
-              keepInPageOnReview: true,
-            },
-          },
-        ),
+        'ui:widget': 'radio',
+        'ui:required': formData => isChapterFieldRequired(formData, 'addChild'),
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-place-of-birth/child-place-of-birth.js]: Just a heads up, changing this to an enum and using radio actually only allows a user to select one. I think we need to keep it how it was originally, possibly use a checkbox widget.

**Review by micahchiang — CHANGES_REQUESTED** [review]
Requesting change for the child options field.

File: src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-place-of-birth/child-place-of-birth.js (hunk 1 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-place-of-birth/child-place-of-birth.js]: Changed this back to a list of checkboxes, I didn't realize that someone needed to be able to pick more than one

**Review by micahchiang — APPROVED** [review]
lgtm

**Pull request merged**